src/utils/Logger.js

```diff
@@ -20,7 +20,7 @@ export default class Logger {
   }
 
   info(msg) {
-    this.log(msg, 'green')
+    this.log(msg, 'red')
```

Comment:
:thinking: `info` type logs sound like they shouldn't have any color, specially not red! The `log` function doesn't support no color at the moment.